infer/lib/audio.py

```diff
@@ -1,18 +1,24 @@
-import platform
+from io import BufferedWriter, BytesIO
+from pathlib import Path
+from typing import Any, Dict
 import ffmpeg
 import numpy as np
 import av
 
+video_format_dict: Dict[str, str] = {
+    "m4a": "mp4",
+}
 
-def wav2(i, o, format):
+audio_format_dict: Dict[str, str] = {
+    "ogg": "libvorbis",
+    "mp4": "aac",
+}
+
+def wav2(i: BytesIO, o: BufferedWriter, format: str):
     inp = av.open(i, "r")
-    if format == "m4a":
-        format = "mp4"
+    format = video_format_dict[format]

```

Comment:
I think the dict will panic if the format is not m4a. You can use the `get` grammar and set the default value as `format` itself.

Reply:
Should be fixed by `f145e7c`